configuration/configuration_test.go

```diff
@@ -148,14 +148,14 @@ func TestLoadConfiguration(t *testing.T) {
 			cfg: &Configuration{
 				Mode: Online,
 				Network: &types.NetworkIdentifier{
-					Network:    ethereum.RopstenNetwork,
+					Network:    ethereum.DevNetwork,
```

Comment:
can we add an additional test for devnet?

Reply:
well, depends if NETWORK=TESTNET means devnet or ropsten. there still is a test for ropsten here https://github.com/coinbase/rosetta-ethereum/pull/141/files/1032355c75d5317c6324ee9fdea22fad474a62fb#diff-0aadb475e41b205384efdcf4231633cdd5097bb5b936000d7ce06989c0e41741L92

Reply:
ah maybe you're asking how to enable the dev net? The only way currently is by setting NETWORK=TESTNET. we could also allow NETWORK=DEV or something like that.

Reply:
@xiaying-peng if you tell me what changes you'd like I'll make them. I'm not quite clear on what you'd like. We can leave TESTNET to mean ROPSTEN and add a DEV network, if that is what you want.